INSTRUCTION.md

```diff
@@ -0,0 +1,45 @@
+# Deployment and Validation Instructions
+
+## Prerequisites
+1. Kubernetes cluster set up using Kind.
+2. Replace `<your-dockerhub-username>` with your Docker Hub username.
+
+---
+
+## Steps to Deploy
+
+1. Run the `bootstrap.sh` script:
+```bash
+   ./bootstrap.sh
+```
+2. Verify that resources are created:
+````
+kubectl get statefulsets -n mysql
+kubectl get pods -n mysql
+kubectl get pods -n mateapp
+````
+
+## Validation Steps
+1. Validate MySQL StatefulSet
+   - Ensure 3 replicas are running:
+        ````
+        kubectl get pods -n mysql
+        ````
+   - Validate readiness and liveness:
+        ````
+        kubectl describe pod <pod-name> -n mysql
+        ````
+     
+2. Validate ToDo App
+   - Port-forward the ToDo app
+        ````
+        kubectl port-forward deployment/todoapp 8000:8000 -n mateapp
+        ````  
+   - Access the app at http://localhost:8080
```

Comment:
There is a discrepancy in the port number. The port-forward command uses port 8000, but the instructions mention accessing the app at http://localhost:8080. Ensure consistency in the port number to avoid confusion.